playbooks/amc.yml

```
@@ -91,8 +91,8 @@
       nginx_sites:
       - xqueue
     - { role: 'rabbitmq', rabbitmq_ip: '0.0.0.0' }
-    - notifier
-    - { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
+#    - notifier
+#    - { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
```

Let's get it back? or at least have an open issue in the this repo so we don't loose it.
```suggestion
    - { role: notifier, NOTIFIER_DIGEST_TASK_INTERVAL: "5" }
```